docs/OV_Runtime_UG/supported_plugins/GPU_RemoteTensor_API.md

```diff
@@ -1,104 +1,139 @@
 
 Class and function declarations for the API are defined in the following files:
-* Windows\*: `gpu/gpu_context_api_ocl.hpp` and `gpu/gpu_context_api_dx.hpp`
-* Linux\*: `gpu/gpu_context_api_ocl.hpp` and `gpu/gpu_context_api_va.hpp`
+* Windows\*: `openvino/runtime/intel_gpu/ocl/ocl.hpp` and `openvino/runtime/intel_gpu/ocl/dx.hpp`
+* Linux\*: `openvino/runtime/intel_gpu/ocl/ocl.hpp` and `openvino/runtime/intel_gpu/ocl/va.hpp`
 
-The most common way to enable the interaction of your application with the Remote Blob API is to use user-side utility classes
+The most common way to enable the interaction of your application with the Remote Tensor API is to use user-side utility classes
 and functions that consume or produce native handles directly.
 
 ## Execution Context User-Side Wrappers
 
-GPU plugin classes that implement the `RemoteContext` interface are responsible for context sharing.
+GPU plugin classes that implement the `ov::RemoteContext` interface are responsible for context sharing.
 Obtaining a pointer to a context object is the first step of sharing pipeline objects.
 The context object of the GPU plugin directly wraps OpenCL context, setting a scope for sharing
-`ExecutableNetwork` and `RemoteBlob` objects.
-To create such objects within user context, explicitly provide the context to the plugin using the
-`make_shared_context()` overloaded function. Depending on the platform, the function accepts the
+`ov::CompiledModel` and `ov::RemoteTensor` objects.
+To create such objects within user context, explicitly provide the context to the plugin using constructor for one
+of `ov::RemoteContext` derived classes: `ClContext`, `VAContext`, `D3DContext`.
+Depending on the class and platform, the constructors accept the
 `cl_context` handle, the pointer to the `ID3D11Device` interface, or the `VADisplay` handle, and
-returns a smart pointer to the `RemoteContext` plugin object.
+returns a smart pointer to the `ov::RemoteContext` plugin object.
 
 If you do not provide any user context, the plugin uses its default internal context.
 The plugin attempts to use the same internal context object as long as plugin options are kept the same.
-Therefore, all ExecutableNetwork objects created during this time share the same context.
+Therefore, all `ov::CompiledModel` objects created during this time share the same context.
 Once the plugin options are changed, the internal context is replaced by the new one.
 
-To request the current default context of the plugin, call the `GetDefaultContext()` method of the core engine.
-To request the internal context of the given `ExecutableNetwork`, use the `GetContext()` method.
+To request the current default context of the plugin, call the `ov::Core::get_default_context()` method of the OpenVINO Core.
+To request the internal context of the given `ov::CompiledModel`, use the `ov::CompiledModel::get_context()` method.
```

Comment:
can we try:
1. Provide examples how to create contexts
2. Structure this information to be not a plain text, maybe some lists (e.g. we can create or wrap existing objects, types of supported contexts: VA, DX, OCL)